graf2d/gpad/src/TPad.cxx

```diff
@@ -1225,13 +1225,16 @@ Int_t TPad::DistancetoPrimitive(Int_t px, Int_t py)
 /// Automatic pad generation by division.
 ///
 ///  - The current canvas is divided in nx by ny equal divisions (pads).
-///  - xmargin defines the horizontal spacing around each pad as a percentage of the canvas
-///    width. Therefore, the distance between two adjacent pads along the x-axis is equal
-///    to twice the xmargin value.
-///  - ymargin defines the vertical spacing around each pad as a percentage of the canvas
-///    height. Therefore, the distance between two adjacent pads along the y-axis is equal
-///    to twice the ymargin value.
+///  - xmargin defines the horizontal spacing between each pad as a percentage of the canvas
```

Comment:
This is overwriting the changed I did in the help. It goes back to the old help.

Reply:
I think he is proposing to change behavior in 6.40, and the new behavior matches by chance the old help message